controllers/clusters/datatest/cassandra_v1beta1.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: clusters.instaclustr.com/v1beta1
 kind: Cassandra
 metadata:
   name: cassandra-cluster
+  namespace: default
+  annotations:
+    defaulter: webhook
```

Comment:
Why do we need `defaulter` annotation?

Reply:
Webhooks are not enabled in this integration test, so I simulated the webhook.